src/vm/i386/cgenx86.cpp

```diff
@@ -299,7 +299,12 @@ void TransitionFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
 
     MethodDesc * pFunc = GetFunction();
     _ASSERTE(pFunc != NULL);
+
+#ifdef WIN64EXCEPTIONS
+    UpdateRegDisplayHelper(pRD, 0);
```

Comment:
@janvorli When I set ``Esp`` as ``Caller-SP`` inside ``UnwindStackFrame``, ``JIT.Regression.CLR-x86-JIT.V2.0-Beta2.b415164`` get stuck (discussed in #9366). It is almost same as #9272, but the callee is managed one instead of native one.

Actually, #9366 was an attempt to implement libunwind-like unwinder without affecting x86/Windows.